acpi/src/lib.rs

```diff
@@ -240,7 +240,7 @@ where
     fn tables_phys_ptrs(&self) -> TablesPhysPtrsIter<'_> {
         // SAFETY: The virtual address of the array of pointers follows the virtual address of the table in memory.
         let ptrs_virt_start = unsafe { self.mapping.virtual_start().as_ptr().add(1).cast::<u8>() };
-        let ptrs_bytes_len = self.mapping.region_length() - mem::size_of::<SdtHeader>();
+        let ptrs_bytes_len = self.mapping.length as usize - mem::size_of::<SdtHeader>();
```

Comment:
I cannot find a `length` field on `PhysicalMapping`. Do you mean to use another field? In any case, my understanding was that, while `mapped_length` may be larger than the number of bytes the caller of `AcpiHandler::map_physical_region` requested, the `region_length` argument pased to `map_physical_region` must persist in the returned `PhysicalMapping`'s field of the same name so that the original code is valid. This is not mentioned in the documentation (or I missed it), though, so I suppose it is purely an assumption.

Reply:
ah, good point.  In my implementation, I had reversed the meanings of region_length and mapped_length due to my earlier uncertainty about the definitions.   I'll close this MR.